content/dstu2/authorization.md

```diff
@@ -68,11 +68,21 @@ resources and exchange context information with client applications.
 
 ## Registration ##
 In order for your client application to utilize any protected resources, 
-your client application must first register using our 
-[code console][CERNER-CODE-CONSOLE]. A Cerner Care account is required to 
+your client application must first register. A Cerner Care account is required to 
 register applications. Obtaining a Cerner Care account is free and 
 requires nothing but an email address.
 
+If registering an application that will not be using offline_access, and that is not 
+going to be accessing services on behalf of a system (is not a confidential client), 
+your client application must first register using our 
+[code console][CERNER-CODE-CONSOLE]. 
+
+If registering a confidential client application (required for offline_access and
+access on behalf of a system), please first request an account via the **sandbox** instance of 
+[Cerner Central System Account Management][SYSTEM-ACCOUNTS]. Once the request is completed,
```

Comment:
Should switch this over to the specific registration page: https://wiki.ucern.com/display/CernerCentral/Requesting+A+System+Account